modules/oe_whitelabel_list_pages/oe_whitelabel_list_pages.info.yml

```diff
@@ -6,9 +6,9 @@ core_version_requirement: ^10
 
 dependencies:
   - extra_field:extra_field
-  - oe_list_pages:oe_list_page_content_type
-  - oe_list_pages:oe_list_pages
+  - oe_content_extra:oe_content_extra_list_pages
+  - oe_whitelabel:oe_whitelabel_helper
```

Comment:
dont add oe_whitelabel:oe_whitelabel_helper dep as this does only declare the content banner view display, no reference to the block is present